Description: Install check-plugin.py as a script
 The installer installs check-plugin.py as hp-check-plugin, do the same from the
 Makefile.
Author: Julian Andres Klode <jak@debian.org>
Origin: vendor
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile.am
+++ b/Makefile.am
@@ -162,10 +162,10 @@ cmddir = $(hplipdir)
 dist_cmd_SCRIPTS = align.py info.py print.py toolbox.py clean.py colorcal.py unload.py testpage.py makeuri.py check.py fab.py levels.py \
 	sendfax.py setup.py makecopies.py probe.py timedate.py firmware.py scan.py systray.py plugin.py linefeedcal.py pqdiag.py \
 	faxsetup.py devicesettings.py printsettings.py query.py pkservice.py wificonfig.py diagnose_plugin.py uninstall.py \
-	upgrade.py config_usb_printer.py diagnose_queues.py logcapture.py doctor.py
+	upgrade.py config_usb_printer.py diagnose_queues.py logcapture.py doctor.py check-plugin.py
 
 homedir = $(hplipdir)
-dist_home_DATA =  check-plugin.py 
+dist_home_DATA =
 dist_home_DATA +=  hplip_clean.sh
 
 # base
